tests/test_commit_message.py

```diff
@@ -85,15 +85,15 @@ def test_commit_message_blocking_reviewers_requested(reviewer_text):
         "r?#group1",
         "r?#group1, #group2",
         "r?reviewer_one,#group1",
-        "r?#group1 r?reviewer_two",
+        "r?#group1 r?reviewer.two",
```

Comment:
The bug report mentions how a single period messed up the parsing of the text between two names.  This test data covers the single period case, but you may want to consider some test data that parses multiple names with periods, or a single name with multiple periods, depending on how the underlying parser goes about its job.